<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlashcardResult</class>
 <widget class="QWidget" name="FlashcardResult">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="menuButton">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>630</y>
     <width>301</width>
     <height>86</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/bgflashc/btnmenu.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="correctLabel">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>270</y>
     <width>381</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 900 30pt &quot;MS UI Gothic&quot;;</string>
   </property>
   <property name="text">
    <string>Correct    : X</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="incorrectLabel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>440</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 900 30pt &quot;MS UI Gothic&quot;;</string>
   </property>
   <property name="text">
    <string>Incorrect    : Y</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="backsc">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/bgflashc/bgscore.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>backsc</zorder>
  <zorder>menuButton</zorder>
  <zorder>correctLabel</zorder>
  <zorder>incorrectLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
